lib/fluent/plugin/in_http.rb

```diff
@@ -512,6 +512,14 @@ def send_response_nobody(code, header)
         data << "\r\n"
         write data
       end
+
+      def include_cors_allow_origin
+        if @cors_allow_origins.include?(@origin)
+          return true
+        end
+        filtered_cors_allow_origins = @cors_allow_origins.select {|origin| origin != ""}
+        return filtered_cors_allow_origins.find {|origin| (start_str,end_str) = origin.split("*",2); @origin.start_with?(start_str) and @origin.end_with?(end_str)} != nil
```

Comment:
Could you separate this one liner block into multiple lines?
By default, fluentd code doesn't use `;` to force one line.

Reply:
Thanks for your review. 
The contents pointed out were fixed. Do I need to `git rebase -i` ?